test/integration/targets/pip/tasks/main.yml

```diff
@@ -18,6 +24,10 @@
       when: ansible_distribution not in ["MacOSX", "Alpine"]
       register: git_install
 
+    - name: ensure wheel is installed
+      pip:
+        name: wheel
```

Comment:
Should this be a specific version of `wheel` to avoid spontaneous test breakage?